apx/common/src/apx_nodeManager.c

```diff
@@ -543,9 +543,13 @@ static void apx_nodeManager_createNode(apx_nodeManager_t *self, const uint8_t *d
                         {
                            APX_LOG_ERROR("[APX_NODE_MANAGER] createOutInitDataFromDefinition failed");
                         }
-                        apx_nodeData_setNodeInfo(nodeData, nodeInfo);
-                        apx_fileManager_sendFileOpen(fileManager, outDataFile->fileInfo.address);
                      }
+                     else

```

Comment:
I have a hard time understanding the consequence of this change. Can we please meet up in person to discuss?